### PR TITLE
Documentation updates and fix code inconsistency

### PR DESCRIPTION
**Pull request by boiarqin**

### CODE REVIEW

File: docs/rules/ternary-parentheses.md [hunk 1]
```diff
@@ -1 +1,37 @@
-# Require Ternary Expressions to have condition enclosed by parentheses (ternary-parentheses)
\ No newline at end of file
+# Require ternary expressions to have condition enclosed by parentheses (ternary-parentheses)
+
+This addresses a stylistic issue. Surrounding the condition in parentheses promotes faster scanning of code and identification of the condition within a ternary expression.
+
+```js
+var qux = (foo) ? bar : baz;
```
Comment (reintroducing) [inline docs/rules/ternary-parentheses.md]: const everywhere in this file, who uses var anyway?

File: tests/lib/rules/ternary-parentheses.js [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
+'use strict';
 
 //------------------------------------------------------------------------------
 // Requirements
 //------------------------------------------------------------------------------
 
-const rule = require("../../../lib/rules/ternary-parentheses"),
-    RuleTester = require("eslint/lib/testers/rule-tester");
+const rule = require('../../../lib/rules/ternary-parentheses'),
+    RuleTester = require('eslint/lib/testers/rule-tester');
```
Comment (reintroducing) [inline tests/lib/rules/ternary-parentheses.js]: separate const definitions with semicolons at end of each line.

**Review by reintroducing — CHANGES_REQUESTED** [review]

File: tests/lib/rules/ternary-parentheses.js [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+        'var o = (a\n && z)\n ? b\n : c;',
+        'var o = (a && y || z) ? b : c;',
+        'var o = (a && (y || z))\n ? b\n : c;'
     ],
     invalid: [
         {
-            code: "var o = a ? b : c;",
-            errors: [{ message: "Expected parentheses around condition of ternary expression." }]
+            code: 'var o = a ? b : c;',
+            errors: [{ message: 'Expected parentheses around condition of ternary expression.' }]
```
Comment (reintroducing) [inline tests/lib/rules/ternary-parentheses.js]: oh and also these spaces after and before {}

File: tests/lib/rules/ternary-parentheses.js (hunk 2 above)
Comment (boiarqin) [reply tests/lib/rules/ternary-parentheses.js]: updated

File: tests/lib/rules/ternary-parentheses.js (hunk 3 above)
Comment (boiarqin) [reply tests/lib/rules/ternary-parentheses.js]: updated

File: docs/rules/ternary-parentheses.md (hunk 1 above)
Comment (boiarqin) [reply docs/rules/ternary-parentheses.md]: Would you actually prefer const here instead of var? Even though this is documentation and not the actual code?

Comment (reintroducing) [reply docs/rules/ternary-parentheses.md]: yes, because we dont use var anywhere anymore. either const or let.

Comment (boiarqin) [reply docs/rules/ternary-parentheses.md]: ok, I updated the test to reflect that as well.

Comment (reintroducing) [reply docs/rules/ternary-parentheses.md]: very nice, i like (in Borat voice, of course).

**Review by reintroducing — APPROVED** [review]

**Pull request merged**